doc/release-process.md

```diff
@@ -28,10 +28,9 @@ Check that dependencies are up-to-date or have been postponed:
 $ ./qa/zcash/updatecheck.py
 ```
 
-If you are missing the `.updatecheck-token` file required to run this script,
-please ask Taylor or another Zcash developer for a copy, or create an
-unprivileged personal access token for a github account and save it to the
-file in the format `username:hex-token`.
+You can optionally create an `$XDG_DATA_HOME/zcash/updatecheck/token` file to
+avoid running into GitHub rate limiting. Create an unprivileged personal access
+token on GitHub and copy the value into the file.
```

Comment:
```suggestion
You can optionally create a file `~/.local/share/zcash/updatecheck/token` (or
`$XDG_DATA_HOME/zcash/updatecheck/token` if the `XDG_DATA_HOME` environment
variable is set) to avoid running into GitHub rate limiting. Create an
unprivileged personal access token on GitHub and copy the value into the file.
```